lib/index.js

```diff
@@ -62,22 +63,21 @@ function plugin(opts){
     throw new Error('"engine" option required');
   }
 
-  // Throw an error for unsupported engines or typos
-  if (!consolidate[opts.engine]) {
-    throw new Error('Unknown template engine: "' + opts.engine + '"');
-  }
-
-  if (typeof opts.exposeConsolidate === 'function') {
-    opts.exposeConsolidate(consolidate.requires)
+  // An renderer for the layout can be added
+  var render;
+  if (typeof opts.renderer === 'function') {
```

Comment:
This opts needs testing.
I would suggest moving the setting of the opts into its own function.
what are your thoughts @ismay